src/libPMacc/include/eventSystem/tasks/TaskGetCurrentSizeFromDevice.hpp

```diff
@@ -1,5 +1,6 @@
 /**
- * Copyright 2013-2015 Felix Schmitt, Rene Widera, Benjamin Worpitz
+ * Copyright 2013-2015 Felix Schmitt, Rene Widera, Benjamin Worpitz,
+ * Alexander Grund
```

Comment:
Could you please indent your name to align with the first name in the first line?